src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
+                        "pselect6\0"
+                        "pwrite64\0"
+                        "pwritev\0"
+                        "read\0"
+                        "readahead\0"
+                        "readdir\0"
+                        "readlink\0"
+                        "readlinkat\0"
+                        "readv\0"
+                        "removexattr\0"
+                        "rename\0"
+                        "renameat2\0"
+                        "renameat\0"
+                        "rmdir\0"
+                        "select\0"
+                        "setxattr\0"
+                        "splice\0"
+                        "stat64\0"
+                        "stat\0"
+                        "statfs64\0"
+                        "statfs\0"
+                        "symlink\0"
+                        "symlinkat\0"
+                        "sync\0"
+                        "sync_file_range2\0"
+                        "sync_file_range\0"
+                        "syncfs\0"
+                        "tee\0"
+                        "truncate64\0"
+                        "truncate\0"
+                        "umask\0"
+                        "unlink\0"
+                        "unlinkat\0"
+                        "userfaultfd\0"
+                        "ustat\0"
+                        "utime\0"
+                        "utimensat\0"
+                        "utimes\0"
+                        "vmsplice\0"
+                        "write\0"
+                        "writev\0",
+
+                        /* Process control, execution, namespaces */
+                        "@Process",
+                        "arch_prctl\0"
+                        "clone\0"
+                        "execve\0"
+                        "execveat\0"
+                        "fork\0"
+                        "kill\0"
+                        "prctl\0"
+                        "setns\0"
+                        "tgkill\0"
+                        "tkill\0"
+                        "unshare\0"
+                        "vfork\0",
+
+                        /* Unusual or obsolete */
+                        "@Special",
+                        "_sysctl\0"
+                        "get_kernel_syms\0"
+                        "get_mempolicy\0"
+                        "modify_ldt\0"
+                        "pciconfig_iobase\0"
+                        "perf_event_open\0"
+                        "query_module\0"
+                        "rtas\0"
+                        "s390_pci_mmio_read\0"
+                        "s390_pci_mmio_write\0"
+                        "s390_runtime_instr\0"
+                        "sgetmask\0"
+                        "ssetmask\0"
+                        "subpage_prot\0"
+                        "switch_endian\0"
+                        "sys_debug_setcontext\0"
+                        "uselib\0"
+                        "vm86\0"
+                        "vm86old\0",
```

Comment:
Hmm, I'd really call this "@obsolete" I must say. And I think stuff like the other `pciconfig_xyz` stuff belongs in this too?